catalog/resource_online_table.go

```diff
@@ -17,7 +17,7 @@ import (
 const onlineTableDefaultProvisionTimeout = 45 * time.Minute
 
 func waitForOnlineTableCreation(w *databricks.WorkspaceClient, ctx context.Context, onlineTableName string) error {
-	return retry.RetryContext(ctx, lakehouseMonitorDefaultProvisionTimeout, func() *retry.RetryError {
+	return retry.RetryContext(ctx, qualityMonitorDefaultProvisionTimeout, func() *retry.RetryError {
```

Comment:
looks like a copy paste error

```suggestion
	return retry.RetryContext(ctx, onlineTableDefaultProvisionTimeout, func() *retry.RetryError {
```